cmd/xl-v1-object.go

```diff
@@ -1020,3 +1029,16 @@ func (xl xlObjects) ListObjectsV2(ctx context.Context, bucket, prefix, continuat
 	}
 	return listObjectsV2Info, err
 }
+
+func (xl xlObjects) getMRFUploadCh() chan partialUpload {
```

Comment:
```suggestion
func (xl xlObjects) getMRFUploadCh() <-chan partialUpload {
```
It can always be changed back and this conveys some intention.